.github/pull_request_template.md

```diff
@@ -1,3 +1,37 @@
+<!--
+
+Hello! Thank you for submitting a docs PR to terraform.io! Feel free to delete
+this message.
+
+- For advice or edits from a tech writer or education engineer,
+  please request review from the "hashicorp/terraform-education" GitHub team.
+
+- When updating screenshots of a web UI, please try to capture
+  the full width of the page, with the viewport size set to 1024px wide.
+
+- If you're a HashiCorp employee with permission to merge to this repo,
+  please get an approving review before merging your own PRs. (If you got
+  review approval on the private fork, that's fine too; just announce it in a
+  comment before merging!)
+
+- To learn more about how the website is built and deployed, how to preview your
+  changes, which content lives where, and more, check out the README.md in the
+  root of this repo.
+
+- When in doubt, ask in the #proj-terraform-docs channel.
+
+-->
+
+## What's this PR about?
+
+<!-- (Delete any that don't apply, add anything you want to.) -->
+
+- Fixing inaccurate docs
+- Making some docs easier to understand
+- Adding/updating docs for new feature
+- Changing behavior or layout of website
```

Comment:
```suggestion
- [ ] Fixing inaccurate docs
- [ ] Making some docs easier to understand
- [ ] Adding/updating docs for new feature
- [ ] Changing behavior or layout of website
```

So it'll be checkboxes that PR owner can check.

Reply:
Ahhhh smart!